docs/adr/0004-return-uninflected-va-keys.md

```diff
@@ -0,0 +1,25 @@
+# 4. Return Uninflected VA Keys in vets-api
+
+Date: 2020-10-30
+
+Whom:
+- Philip Becker
+
+## Status
+
+Accepted
+
+## Context
+
+Using the Rails 6 zeitwerk file loader requires strict naming conventions of files.  So without a Rails inflection, `va_something` class files are expected to have the class name of `VaSomething`.  With an inflection, that file would be the expected `VASomething`.  Using this inflection has a downstream affect on json responses transformed into camelCase by the [olive_branch](https://github.com/vigetlabs/olive_branch) gem.
+
+## Decision
+
+To create a deprecation path, there will be a patch implemented for the olive_branch middleware that returns an uninflected version of keys containing `VA`.
+
+## Consequences
+
+- We will be returing additional keys in JSON responses using a `VA`-named key, so this will make payload sizes slightly larger.
```

Comment:
```suggestion
- We will be returning additional keys in JSON responses using a `VA`-named key, so this will make payload sizes slightly larger, supporting both naming schemes (`VA`-named key and `Va`-named key)
```